### ISSUE TITLE
Display errors in the UI

### ISSUE THREAD

**Issue by felixbade** [felixbade/transcribe#5]
At the moment the errors only go to the console and the UI says ”Transcribing...” forever.

**Comment by felixbade** [comment]
This is annoying because sometimes the transcribing *does* take a long time.

**Comment by ontl** [comment]
Yeah I tried submitting an mp3 for translation, but it was over-quota for the key I was using (in fact, I hadn't properly enabled billing for that key).

No errors displayed in UI, but Console says:

`Failed to load resource: the server responded with a status of 429 ()`

That results in `segments` being undefined when we try to console.log it at line 72 of index.js, and also segments then not being iterable on line 76.

Would recommend passing along the error in the UI.

**Comment by felixbade** [comment]
Oh… I’ll remove the claim ”no quota” from README!

**Comment by felixbade** [comment]
Example situation, pretty common.
<img width="1079" alt="Screenshot 2023-05-11 at 14 34 57" src="https://github.com/felixbade/transcribe/assets/6554998/51232691-4e83-4ecd-b96d-06f6469968f4">
